packages/sx.js/src/clients/offchain/types.ts

```diff
@@ -21,6 +21,16 @@ export type StrategyConfig = {
   metadata?: Record<string, any>;
 };
 
+export type SnapshotInfo = {
+  at: number | null;
+  chainId?: number;
+};
```

Comment:
Are we fine with this being duplicate of 

https://github.com/snapshot-labs/sx-monorepo/blob/9ff0d54209bca0c3cfc61d659138b1ccc1b18a23/apps/ui/src/networks/types.ts#L66-L69

Reply:
I think so, I duplicated because it didn't feel like complex enough (and generic enough - it's only used for offchain client) to be imported/shared. If we move all networks to use same type then probably it would make sense.